component/index.android.js

```diff
@@ -47,6 +47,12 @@ NotificationsComponent.prototype.clearLocalNotification = function(details, tag)
 NotificationsComponent.prototype.cancelAllLocalNotifications = function() {
 	RNPushNotification.cancelAllLocalNotifications();
 };
+NotificationsComponent.prototype.openAlarmPermissionSettings = function() {
+	RNPushNotification.openAlarmPermissionSettings();
+};
+NotificationsComponent.prototype.canScheduleExactAlarms = function() {
+	return RNPushNotification.canScheduleExactAlarms();
```

Comment:
FYI this is currently returning undefined. I haven't found out why yet

Reply:
@waelvilla Is this now what you mentioned is fixed?

Reply:
Yes, this fixed now